spec/models/post_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'should be valid' do
```

Comment:
## [OPTIONAL] 

- [ ]  Kindly change this message to `should not be valid` as is testing the invalidity of the post.